Tests/MongoSwiftSyncTests/SyncChangeStreamTests.swift

```diff
@@ -203,7 +206,7 @@ internal struct ChangeStreamTest: Decodable, FailPointConfigured {
 
         if let expectations = self.expectations {
             let commandEvents = monitor.commandStartedEvents()
-                .filter { $0.commandName != "isMaster" }
+                .filter { !["isMaster", "killCursors"].contains($0.commandName) }
```

Comment:
SPEC-1505 added language to the spec indicating `killCursors` events should be ignored:
> For each `(expected, idx)` in expectations - If `actual[idx]` is a `killCursors` event, skip it and move to `actual[idx+1]`. - Else assert that `actual[idx]` MATCHES `expected`

(in https://github.com/mongodb/specifications/blob/master/source/change-streams/tests/README.rst#spec-test-runner)